app/models/resource.rb

```diff
@@ -2,7 +2,7 @@ class Resource < ApplicationRecord
   acts_as_bookable time_type: :range,
                    capacity_type: :closed
 
-  def current_day_bookings
-    self.bookings.select {|booking| booking.time_start.to_date == Date.today}
+  def current_day_bookings(date)
+    self.bookings.select {|booking| booking.time_start.to_date == Date.parse(date)}
```

Comment:
Look @MikaelFeher - an actual use of the `select` method. There IS a reason we bring that up in the Library Challenge ;-)